ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -711,7 +711,7 @@ retry=0
 while [[ \${ran_cmds} -eq 0 ]] ; do
 sync
 sleep 4
-cli_entry=(\$(cli_client -qc | grep "fn_id: 0x4 .* vport_id \${ACC_VPORT_ID}" | sed 's/: / /g' | sed 's/addr //g'))
+cli_entry=(\$(cli_client -qc | grep "fn_id: 0x4 .* vport_id \${ACC_VPORT_ID} " | sed 's/: / /g' | sed 's/addr //g'))
```

Comment:
can we use grep -w for the exact match instead of " " after acc_vport_id?.

Reply:
done